pkgs/development/python-modules/sshpubkeys/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, ecdsa, cryptography }:
+
+buildPythonPackage rec {
+  pname = "sshpubkeys";
+  version = "3.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "105g2li04nm1hb15a2y6hm9m9k7fbrkd5l3gy12w3kgcmsf3k25k";
+  };
+
+  propagatedBuildInputs = [
+    ecdsa
+    cryptography
+  ];
+
+  # ModuleNotFoundError: No module named 'tests'
+  doCheck = false;
```

Comment:
You might have to add `PYTHONPATH=".:$PYTHONPATH"`.

Reply:
```
======================================================================
ERROR: tests (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests
Traceback (most recent call last):
  File "/nix/store/ydk0mfpvn9smcmn72wc9i20slv1d2b79-python3-3.7.2/lib/python3.7/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
ModuleNotFoundError: No module named 'tests'
```

Reply:
Setting explicitly in the derivation, `PYTHONPATH = ".:$PYTHONPATH";` errors with 
```
Found duplicated packages in closure for dependency 'sshpubkeys':
  sshpubkeys 3.1.0 (/nix/store/2vd539zsgavw0c7vajsy4a18wwp6hcsi-python2.7-sshpubkeys-3.1.0/lib/python2.7/site-packages)
  sshpubkeys 3.1.0 (/build/sshpubkeys-3.1.0)
```

Reply:
Tried including that PR as a patch but didn't seem to work, just changed it to use the Github source since the `tests/` directory is there.

Reply:
Please include a comment with reason for not using `fetchPyPI`.